core/eolearn/core/core_tasks.py

```diff
@@ -75,7 +75,7 @@ def filesystem(self):
         """ A filesystem property that is being lazy-loaded the first time it is needed
```

Comment:
It would be nice to also update the description that it is no longer lazy-loaded

Reply:
I completely agree and fixed it.